java-plugin/src/main/java/fr/greencodeinitiative/java/checks/OptimizeImageVectorsSize.java

```diff
@@ -0,0 +1,161 @@
+package fr.greencodeinitiative.java.checks;
```

Comment:
indeed, this new rule implementation isn't included in ecoCode plugin because there is no declaration in `RulesList` class : this is mandatory to include it.